Generals/Code/GameEngineDevice/Source/Win32Device/Common/Win32LocalFileSystem.cpp

```diff
@@ -69,6 +68,9 @@ File * Win32LocalFileSystem::openFile(const Char *filename, Int access /* = 0 */
 		}
 	}
 
+	// TheSuperHackers Mauller 21/04/2025 Only create the new file instance when it is necassary or we leak file handles!
+	Win32LocalFile *file = newInstance( Win32LocalFile );	
```

Comment:
This commit can be combined with the other commit that does the same kind of fix with ArchiveFile.